libraries/@mattermost/rnutils/ios/RNUtilsWrapper.swift

```diff
@@ -251,6 +252,14 @@ import React
             group.wait()
             return dimensions
     }
+
+    @objc public func setHasRegisteredLoad() {
+        hasRegisteredLoad = true
+    }
+
+    @objc public func getHasRegisteredLoad() -> Dictionary<String, Any> {
+        return ["hasRegisteredLoad": hasRegisteredLoad]
```

Comment:
Q: noticed that this is an object, would it be easier to just pass the boolean directly?

Reply:
It would be easier, but I had problems making it work with Booleans. Not familiar enough with the native side of things, and could not find examples where we return a single Boolean value.

Reply:
No big deal, thought maybe we want to always send object vs a single primitive.